src/main/java/ch/njol/skript/expressions/ExprArmorSlot.java

```diff
@@ -28,75 +27,114 @@
 	"<li>Horses: Horse armour (doesn't work on zombie or skeleton horses)</li>",
 	"<li>Wolves: Wolf Armor</li>",
 	"<li>Llamas (regular or trader): Carpet</li>",
-	"</ul>"
+	"</ul>",
+	"Saddle is a special slot that can only be used for: pigs, striders and horse types (horse, camel, llama, mule, donkey)."
 })
 @Examples({
 	"set chestplate of the player to a diamond chestplate",
 	"helmet of player is neither tag values of tag \"paper:helmets\" nor air # player is wearing a block, e.g. from another plugin"
 })
 @Keywords("armor")
-@Since("1.0, 2.8.0 (armor), 2.10 (body armor)")
+@Since("1.0, 2.8.0 (armor), 2.10 (body armor), INSERT VERSION (saddle)")
 public class ExprArmorSlot extends PropertyExpression<LivingEntity, Slot> {
 
-	private static final Set<Class<?>> bodyEntities = new HashSet<>(Arrays.asList(Horse.class, Llama.class, TraderLlama.class));
+	private static final Set<Class<? extends Entity>> bodyEntities =
+		new HashSet<>(Arrays.asList(Horse.class, Llama.class, TraderLlama.class));
+	private static final Set<Class<? extends Entity>> saddleEntities =
+		new HashSet<>(Arrays.asList(Pig.class, Strider.class, AbstractHorse.class));
```

Comment:
While we're here, I would update the naming convention (`SCREAMING_SNAKE_CASE`) and also use `Set.of(<classes>)` for simplified construction